.github/workflows/main.yml

```diff
@@ -0,0 +1,61 @@
+name: HTTPStan
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - '**'
+
+jobs:
+  tests:
+    name: HTTPStan tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-10.15]
+        python-version: [3.7, 3.8]
+      fail-fast: false
+    steps:
+      - name: Checkout github
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install poetry
+        run: |
+          pip install -U pip
```

Comment:
```suggestion
```
This is not part of travis-ci. If failing to upgrade pip is causing problems, let's discuss it in an issue. From my perspective, if upgrading pip is required to install httpstan, then we need to update the instructions. Fortunately, upgrading pip is not required right now.

Reply:
Yes, we can remove this